martin/src/cog/errors.rs

```diff
@@ -52,4 +52,9 @@ pub enum CogError {
         "The pixel size of the image {0} is not squared, the x_scale is {1}, the y_scale is {2}"
     )]
     NonSquaredImage(PathBuf, f64, f64),
+
+    #[error(
+        "Get origin failed for {0}: the length of ModelTiepointTag should be >= 6, or the length of ModelTransformationTag should be >= 12"
+    )]
+    GetOriginFailed(PathBuf),
```

Comment:
This requiring another server major bump is another case where decoupling martin and it's libs semver would be handy.

Regarding this: all error types should likely be marked as no eyhausive.